rero/rero-ils
# notification: circ policies new parameters

* Adds notification parameters to circulation policy.
* number_of_days_before_due_date: needed for the due_soon notification.
* Rnumber_of_days_after_due_date: needed for the over_due first_reminder notification.

Co-Authored-by: Aly Badr <aly.badr@rero.ch>
Co-Authored-by: Peter Weber <peter.weber@rero.ch>

# To test:
* `./scripts/bootstrap`
* `./scripts/setup`
* edit one of the policies at `http://localhost:5000/admin/records/circ_policies` and check out the new parameters.
